src/services/date-utilities.ts

```diff
@@ -360,4 +360,45 @@ export default class DateUtilities {
   static isDateFuture (date: FormattedDateTimeGmt): boolean {
     return (new Date(date) > new Date())
   }
+
+  /**
+   * Add a number of months to a date and return "YYYY-MM-DD".
+   * Date must be in the format of "YYYY-MM-DD" and months is a number
+   * @example (3, 2023-02-03) -> "2023-05-03"
+   * @example (18, 2023-02-03) -> "2024-08-03"
+   */
+  static addMonthsToDate (month: number, date: string): string {
+    if (!date) {
+      date = '2023-01-01'
+    }
+    const temp = this.yyyyMmDdToDate(date)
+    temp.setMonth(temp.getMonth() + month)
+    const dateAfterAddition = this.dateToYyyyMmDd(temp)
+    return dateAfterAddition
+  }
+
+  /**
+   * Decrease one date from another and return number of months as the difference.
+   * Dates must be in the "YYYY-MM-DD" format
+   * @example (2023-02-03, 2024-08-03) -> 18
+   * @example (2023-02-03, 2023-04-03) -> 2
+   */
+  static subtractDates (dateFrom: string, dateTo: string): number {
+    if (!dateFrom) {
+      dateFrom = '2023-01-01'
+    }
+    if (!dateTo) {
+      dateTo = this.addMonthsToDate(24, dateFrom)
+    }
```

Comment:
I recommend against this. I'd like to see this method be "pure", and leave the fallback logic to whoever calls it.

Reply:
Fixed, thanks.